Dockerfile.assisted-installer.ocp

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.18 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 AS builder
```

Comment:
Our go.mod is still `1.21` .. I assume that doesn't cause a problem for you guys?